### PR TITLE
DOCS-10985-dw-multipart-sample-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Added iterate multipart example

### CODE REVIEW

File: modules/ROOT/pages/dataweave-formats-multipart.adoc [hunk 1]
```diff
@@ -426,6 +427,54 @@ Content-Disposition: form-data; name="otherJsonPart"
...
+}
+--34b21
+Content-Disposition: form-data; name="file2"; filename="a.html"
+Content-Type: text/html
+
+<!DOCTYPE html>
+<title>
+Available for download!
+</title>
+-34b21-'
```
Comment (leansh) [inline modules/ROOT/pages/dataweave-formats-multipart.adoc:462]:
```suggestion
--34b21--'
```

File: modules/ROOT/pages/dataweave-formats-multipart.adoc [hunk 2]
```diff
@@ -426,6 +427,54 @@ Content-Disposition: form-data; name="otherJsonPart"
...
+var multi = '--34b21
+Content-Disposition: form-data; name="text"
+Content-Type: text/plain
+
+Book
+--34b21
+Content-Disposition: form-data; name="file1"; filename="a.json"
+Content-Type: application/json
+
+{{{}}
```
[the remark this answers is not in the document]
Comment (leansh) [reply modules/ROOT/pages/dataweave-formats-multipart.adoc:449]:
```suggestion
{
```

**Review by leansh — CHANGES_REQUESTED** [review]

**Review by leansh — APPROVED** [review]

File: modules/ROOT/pages/dataweave-formats-multipart.adoc [hunk 3]
```diff
@@ -426,6 +427,54 @@ Content-Disposition: form-data; name="otherJsonPart"
 ------myboundary------
 ----
 
+[[example4]]
+=== Example: Iterate through Multipart payload
+
+This example shows how DataWeave iterates through through all the parts of the payload and process the message individually.
```
Comment (dukesphere) [inline modules/ROOT/pages/dataweave-formats-multipart.adoc:433]:
```suggestion
This example shows how DataWeave iterates through all the parts of the payload and processes the each part individually.
```

Comment (dukesphere) [reply modules/ROOT/pages/dataweave-formats-multipart.adoc:433]: "and processes the message individually" is unclear. I assume it means "each part of the message"? I'm not sure but that's what I suggested above.

**Review by dukesphere — COMMENTED** [review]
suggestion

**Review by dukesphere — APPROVED** [review]

**Pull request merged**